TestFoundation/TestProcess.swift

```diff
@@ -565,6 +565,25 @@ class TestProcess : XCTestCase {
         XCTAssertEqual(String(data: stdoutData, encoding: .utf8)?.trimmingCharacters(in: .whitespacesAndNewlines), "No files specified.")
     }
 
+    func test_fileDescriptorsAreNotInherited() throws {
```

Comment:
This needs to be conditionalized to !os(Windows).

Reply:
done